README.md

```diff
@@ -64,27 +58,34 @@ We are active and enthusiastic participants of the following community support c
-| [@taquito/utils](packages/taquito-utils)                         | Provides different encoding and decoding utilities            |
-| [@taquito/tzip12](packages/taquito-tzip12)                       | TZIP-12 allows retrieving NFT/token metadata                  |
-| [@taquito/tzip16](packages/taquito-tzip16)                       | TZIP-16 allows retrieving contract metadata and executing off-chain views |
-| [@taquito/beacon-wallet](packages/taquito-beacon-wallet)         | TZIP-10 implementation of a Wallet API                        |
+| Low-Level Packages                                               | Responsibility                                                                                       |
+| ---------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| [@taquito/local-forging](packages/taquito-local-forging)         | Local "forging": serialization of Tezos operations as bytes                                          |
+| [@taquito/michelson-encoder](packages/taquito-michelson-encoder) | Creates JS abstractions of Smart Contracts                                                           |
+| [@taquito/michel-codec](packages/taquito-michel-codec)           | Converts Michelson between forms, expands Macros, etc                                                |
+| [@taquito/remote-signer](packages/taquito-remote-signer)         | Provides the facility to use a remote signer, such as https://signatory.io                           |
+| [@taquito/rpc](packages/taquito-rpc)                             | RPC client library: every rpc endpoint has its own method                                            |
+| [@taquito/signer](packages/taquito-signer)                       | Provides functionality to sign data using tezos keys                                                 |
+| [@taquito/utils](packages/taquito-utils)                         | Provides different encoding and decoding utilities                                                   |
+| [@taquito/tzip12](packages/taquito-tzip12)                       | TZIP-12 allows retrieving NFT/token metadata                                                         |
+| [@taquito/tzip16](packages/taquito-tzip16)                       | TZIP-16 allows retrieving contract metadata and executing off-chain views                            |
+| [@taquito/beacon-wallet](packages/taquito-beacon-wallet)         | TZIP-10 implementation of a Wallet API                                                               |
+| [@taquito/http-utils](packages/taquito-http-utils)               | Provides functionality configure http request options                                                |
+| [@taquito/core](packages/taquito-core)                           | Defined core classes, types and interfaces                                                           |
```

Comment:
I would rephrase to:
'Provides parent and core types, classes, and interfaces for other Taquito packages or external uses'